backend/constants.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
-    FRONTEND_SERVER: 'https://cpsc-455-music-tracker.herokuapp.com',
+    FRONTEND_SERVER: process.env.NODE_ENV === 'production' ? 'https://cpsc-455-music-tracker.herokuapp.com' : 'http://localhost:3000',
     CLIENT_ID: 'f943ad62119e426e878c674c01de2358',
```

Comment:
SHall we change this now

Reply:
ima merge this test that everything works, then change the keys